go/libraries/doltcore/merge/merge_schema.go

```diff
@@ -804,7 +804,7 @@ func mergeChecks(ourChks, theirChks, ancChks schema.CheckCollection) ([]schema.C
 			CheckExpression: chk.Expression(),
 			Enforced:        chk.Enforced(),
 		}
-		colNames, err := sqle.ColumnsFromCheckDefinition(nil, &chkDef)
+		colNames, err := sqle.ColumnsFromCheckDefinition(sql.NewEmptyContext(), &chkDef)
```

Comment:
There should probably be an existing context, probably pass that one in, instead of creating a new one.